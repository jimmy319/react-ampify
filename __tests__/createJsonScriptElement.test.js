const reactAmpify = require('../index.js')
const React = require('react')

test('createJsonScriptElement functionality', () => {
  // arrangement
  const { createJsonScriptElement } = reactAmpify
  const json = {
    'id': 'this is a test',
    'name': 'works like a charm'
  }

  // action
  const expected = createJsonScriptElement(json)

  // assertion
  expect(expected).toMatchSnapshot()
})
