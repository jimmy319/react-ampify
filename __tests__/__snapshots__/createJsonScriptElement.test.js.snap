// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createJsonScriptElement functionality 1`] = `
<script
  dangerouslySetInnerHTML={
    Object {
      "__html": "{\\"id\\":\\"this is a test\\",\\"name\\":\\"works like a charm\\"}",
    }
  }
  type="application/json"
/>
`;
